central/networkgraph/flow/datastore/internal/store/postgres/store.go

```diff
@@ -98,6 +110,10 @@ var (
 	// to deal with failures if we just sent it all.  Something to think about as we
 	// proceed and move into more e2e and larger performance testing
 	batchSize = 10000
+
+	deleteTimeout = 3 * time.Minute
```

Comment:
I believe this is based on your experiments with heavy data loads. You can further split the deletes across smaller chunks using of windows on the `LastSeenTimestamp`, like `orphanWindow - 60`, `orphanWindow - 30`, and `orphanWindow - 1`, so that we stay optimistic about the `deleteTimeout`.